testing/testcafe/tests/editors/button/floatingAction.ts

```diff
@@ -125,18 +111,135 @@ test('FAB with two speed dial action buttons', async (t) => {
 
   await testScreenshot(t, takeScreenshot, 'FAB with two speed dial action buttons.png', {
     element: '#container',
-    shouldTestInCompact: true,
-    compactCallBack: async () => {
+  });
+
+  await t
+    .expect(compareResults.isValid())
+    .ok(compareResults.errorMessages());
+}).before(async () => {
+  await setStyleAttribute(Selector('#container'), 'width: 300px; height: 300px;');
+  await appendElementTo('#container', 'div', 'speed-dial-action');
+  await appendElementTo('#container', 'div', 'speed-dial-action-trash');
+
+  await setGlobalConfig();
+
+  await createWidget('dxSpeedDialAction', {
+    label: 'Add row',
+    icon: 'plus',
+    index: 1,
+    visible: true,
+  }, '#speed-dial-action');
+
+  await createWidget('dxSpeedDialAction', {
+    label: 'Remove Row',
+    icon: 'trash',
+    index: 2,
+    visible: true,
+  }, '#speed-dial-action-trash');
+});
+
+fixture.disablePageReloads`FloatingAction - compact theme`
+  .page(url(__dirname, '../../container.html'));
+
+for (const label of ['Add Row', '']) {
+  for (const icon of ['home', '']) {
+    test(`FAB with one speed dial action button, label: ${label}, icon: ${icon}`, async (t) => {
+      const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+      await t
+        .expect(await takeScreenshot(`FAB with one speed dial action button,label='${label}',icon='${icon}'${getThemePostfix(`${process.env.theme}-compact`)}.png`, '#container'))
+        .ok();
+
+      await t
+        .expect(compareResults.isValid())
+        .ok(compareResults.errorMessages());
+    }).before(async () => {
+      await changeTheme(`${process.env.theme}.compact`);
+
+      await setStyleAttribute(Selector('#container'), 'width: 300px; height: 300px;');
+      await appendElementTo('#container', 'div', 'speed-dial-action');
+
+      await setGlobalConfig();
+
       await ClientFunction(() => {
-        (window as any).DevExpress.ui.repaintFloatingActionButton();
+        (window as any).DevExpress.config({
```

Comment:
Is ClientFunction need to be called here after setGlobalConfig call? It seems they make the same thing